packages/space-opera/editor/index.html

```diff
@@ -89,47 +96,68 @@
     background-color: #202124;  /* GOOGLE_GREY_900 */
     padding: 0;
     width: 360px;
+    min-width: 360px;
+    max-width: 360px;
   }
 
   me-tabs a {
     color: #dadcff; /* GOOGLE_GREY_300, but.. bluer */
   }
 
   me-tabbed-panel {
-    max-height: calc(100vh - 96px);
+    max-height: calc(100vh - 45px);
     overflow-y: auto;
     display: block;
   }
+  .privacy {
+    color: var(--expandable-section-text);
+    margin: 5px 16px;
+    position: absolute;
+    bottom: 0;
+    font-size: small;
+  }
+  .privacy-link {
+    color: var(--expandable-section-text);
+  }
 </style>
 <body>
-  <model-editor>
-    <div class="app">
+<model-editor>
+  <div class="app">
     <div class="editor-body-root">
       <div class="mvContainer">
         <model-viewer-preview id="editing_adapter">
         </model-viewer-preview>
       </div>
       <me-tabs>
-        <me-tabbed-panel icon="save">
-          <model-viewer-snippet></model-viewer-snippet>
+        <me-tabbed-panel icon="import_export">
+          <me-export-panel></me-export-panel>
+          <div class="privacy">
+            This &lt;model-viewer&gt; editor does not send any imported content to servers:
+            <a href="https://policies.google.com/privacy" class="privacy-link">
+              Privacy
+            </a>
+          </div>
         </me-tabbed-panel>
         <me-tabbed-panel icon="create">
+          <me-export-panel header="true"></me-export-panel>
           <me-transformation-controls></me-transformation-controls>
           <me-ibl-selector></me-ibl-selector>
           <me-animation-controls></me-animation-controls>
           <me-hotspot-panel></me-hotspot-panel>
           <me-poster-controls></me-poster-controls>
         </me-tabbed-panel>
-        <me-tabbed-panel icon="color_lens">
-          <me-materials-panel></me-materials-panel>
-        </me-tabbed-panel>
         <me-tabbed-panel icon="photo_camera">
+          <me-export-panel header="true"></me-export-panel>
           <me-camera-settings></me-camera-settings>
         </me-tabbed-panel>
+        <me-tabbed-panel icon="color_lens">
```

Comment:
Can we move the metallicRoughness texture below its factors to match the others?